### PR TITLE
Curve lp tests

### PR DESCRIPTION
**Pull request by Paul4912**

### CODE REVIEW

**Comment by c-n-o-t-e** [comment]
Also can you adopt the prettier format.

File: contracts/peripheral/Strategies/Curve.sol [hunk 1]
```diff
@@ -16,10 +16,16 @@ interface ICurvePool {
 interface ICurvePoolFactory {
     function get_coins(address _pool) external view returns (address[4] memory);
 
-    function deploy_plain_pool(string memory _name, string memory _symbol, address[4] memory _coins, uint256 _A, uint256 _fee) external returns (address);
+    function deploy_plain_pool(
```
Comment (c-n-o-t-e) [inline contracts/peripheral/Strategies/Curve.sol:19]: Can you remove the functions found in lines 19-29 as they're used in the contract.

File: test/debt/IncurDebt.js [hunk 2]
```diff
@@ -792,6 +807,38 @@ describe("IncurDebt", async () => {
                 .connect(daiHolder)
                 .createLP(ohmAmount, daiAmount, uniSwapStrategy.address, data);
         });
+
```
[the remark this answers is not in the document]
Comment (c-n-o-t-e) [reply test/debt/IncurDebt.js:810]: I meant testing for these instances 

```if (poolTokens[0] == ohmAddress) {
            if (poolTokens[1] != pairTokenAddress) revert CurveStrategy_LPTokenDoesNotMatch();
            if (_ohmAmount != amounts[0]) revert CurveStrategy_AmountsDoNotMatch();
            if (_pairTokenAmount != amounts[1]) revert CurveStrategy_AmountsDoNotMatch();
        } else if (poolTokens[1] == ohmAddress) {
            if (poolTokens[0] != pairTokenAddress) revert CurveStrategy_LPTokenDoesNotMatch();
            if (_ohmAmount != amounts[1]) revert CurveStrategy_AmountsDoNotMatch();
            if (_pairTokenAmount != amounts[0]) revert CurveStrategy_AmountsDoNotMatch();
```

File: contracts/peripheral/Strategies/Curve.sol (hunk 1 above)
Comment (Paul4912) [reply contracts/peripheral/Strategies/Curve.sol:19]: i need them for tests

**Pull request merged**